docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.7.1
+
+ARG RAILS_ENV
+ENV RAILS_ENV=$RAILS_ENV
+
+RUN apt-get update -qq && \
+  apt-get install -y --no-install-recommends gnupg2 curl && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
Will changing the RAILS_ENV we use with this image impact the system packages we install?

Reply:
I don't expect it would touch the system packages as RAILS_ENV is only really used to determine which bundle groups to load.